scripts.py

```diff
@@ -41,7 +41,7 @@ def run_script(script, stage, *args):
         raise RuntimeError("Missing interpreter in %s." % script)
     interp = line[2:].split()
     if interp[0] == '/usr/bin/env':
-        if len (interp) < 2 or interp[1] not in ['python3']:
+        if len (interp) < 2 or interp[1] not in ['python', 'python3']:
```

Comment:
While this works, I think it would be better to remove this whole section about checking the interpreter. The kernel will fail to execute the file in the unusual case that it can't be executed which makes it just another list of items to keep up to date.

Reply:
I agree. When I saw that code time ago it was my tough too. Also if we want to replace the script with a binary executable I don't see the problem.

Reply:
Maybe however you would need a `try`/`except` around `util.runCmd2` below

Reply:
Thank all! Interpreter check is removed now.
`util.runCmd2` internal already handled exception, so need not to do again.

Reply:
`util.runCmd2` catches exceptions raised by calling `subprocess.Popen` but then it reraises them so I think this code does need to handle the exception.

Reply:
If captured `util.runCmd2` exception, what actions needs to do
- logging and raise again
- or do not raise, but return `rc, out, err`, while what's the preferred values?

Could you show some pseudo-code to let me know your exact point?

Reply:
It shouldn't raise it again since in general (with the exception of answerfile generators), script failures are not fatal and callers do not expect it. Also, `runCmd2()` already logs the error.

I would suggest doing something like this:

```
try:
    rc, out, err = util.runCmd2(...)
except Exception:
    # match shell error code for exec failure
    return 127, b'', b''
```

Reply:
Fixed.